### ISSUE TITLE
Just a big Thanks!!!! 👍 

### ISSUE THREAD

**Issue by ivanvermeyen** [lorisleiva/laravel-deployer#2]
Hi,

Not really an issue, just wanted to give a big **THANKS** for sharing this! 👍 👍 👍 
I wanted to try Deployer for some time, but never got the time to dig in and write the script.
Until now, I've been using Capistrano, but I will be trying out your deployer soon!

**Comment by preeteshjain** [comment]
Second this. This is a neat tool!

**Comment by lorisleiva** [comment]
Thanks I really appreciate. Let me know your thoughts on the package once you’ve had the chance to use it. 🙏🏻

**Comment by ivanvermeyen** [comment]
I just deployed my personal site and the deploy experience is amazing 🌟 
Very clean output and it **just works** out of the box.

The `deploy:init` command is also very handy.
I just had to manually enter the deploy path, because it defaults to the server name and not the website's domain (= app name?). I'm using a different deploy server name because the site is behind CloudFlare and I have a few sites on the same server. But that's really no biggie!

You made something really really beautiful here!

<img width="688" alt="schermafbeelding 2018-03-14 om 21 34 31" src="https://user-images.githubusercontent.com/3598622/37429619-2a0be9f4-27d0-11e8-9a15-07e0d4fdac52.png">
